### DISCUSSION TITLE
Palette installed: TypeError: Joi$7.string is not a function (line:876)

### DISCUSSION THREAD

**Discussion by grucha532** [zachowj/node-red-contrib-home-assistant-websocket#458] [Q&A]
Hello,

I'm new to node-red so I would be grateful if someone would point me in the right direction if it's an error of a newbie...
I have node-red in a docker container of a RPBi (installed as per [https://nodered.org/docs/getting-started/docker](https://nodered.org/docs/getting-started/docker) ). I've installed the palette as shown below. The nodes were there on the right side panel so I've inserted a node into the flow a tried to configure my HA server. The node settings went on and off (blinking like every second) so I refreshed the browser tab. After that the nodes disappeared from the side panel and I found this error in the settings. I've tried to reinstall the palette/restart RPi few times but it didn't work.

Do you have any idea what's the problem/possible solution for it?

![Capture](https://user-images.githubusercontent.com/64309335/129326464-be4b3d99-5d0e-42c6-b945-fe1ffeb93b10.JPG)

**Comment by zachowj** [comment]
Remove it and try reinstalling it.

**Reply by grucha532** [reply]
Hi, thanks but I've tried it several times... The only solution that worked for me was to reinstall the whole system. This time I've added node-red not as separate container but as an addon.

Thanks anyway :)
